codeship-services.yml

```diff
@@ -36,3 +37,38 @@ app:
         HR_NOTIFICATIONS_EMAIL: hr@example.com
         ABANDONED_USER_bestPracticeUrl: http://www.example.com/best-practices.html
         ABANDONED_USER_deactivateInstructionsUrl: http://www.example.com/deactivate-instructions.html
+        MFA_WEBAUTHN_apiBaseUrl: externalapi:8080/
+        MFA_WEBAUTHN_apiKey: EC7C2E16-5028-432F-8AF2-A79A64CF3BC1
```

Comment:
Are these dummy values or real values? They look real, so it seemed worth asking. If they are dummy values, would it be easy to change some segment of them to make that obvious? (e.g. changing `EC7C2E16` to `AAAAAAAA`)

Reply:
They're real values.

Reply:
If they're real values, do we need to be concerned about them being made public here? (Or by "real values" do you something other than "actual values from some production system"?)

Reply:
They're real but not production system values.  They need to "work" with the api backend.